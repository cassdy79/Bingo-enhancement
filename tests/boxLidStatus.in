1
a
b
1 Y 2
2 Y 5
3 B 3
0 U 6
4 U 6
5 U 6
0 L 6
0 Y 6
save boxLidStatus
4