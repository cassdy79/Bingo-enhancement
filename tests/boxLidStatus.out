a
b
-1
-14
0
F...............
YYBB
RBLL
RRYL
BLLU
RRYL
byrul
lbYru
ulByr
rulby
yrulb
byrul
lbyru
ulbyr
rulby
yrulb
.
..
...
....
.....
.
..
...
....
YYY..
       
       

RRYBYRRLLBUURLYRBRUBLLBBBRRRBRUBRRUBBYLBLBLYULBYYRLUBYYURLUYULUY
23